src/Application/Yarp.Application.csproj

```
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <RuntimeIdentifiers>win-x64;win-arm64;linux-x64;linux-arm64;</RuntimeIdentifiers>
+    <TargetFramework>net8.0</TargetFramework>
```

```suggestion
    <TargetFramework>net9.0</TargetFramework>
```
If we're the only ones expected to build this, can we just always use the latest?

---

We are the only one, but since it will be used in a Dockerfile in another repo, I prefer keeping the target framework explicitely fixed